### ISSUE TITLE
The minus button: too easy to hit 

### ISSUE THREAD

**Issue by AnAmelie** [olivierkes/manuskript#841]
I use git to backup my writing projects so this isn't a huge deal for me, but sometimes it's annoying. 

Sometimes the plus and minus buttons work too well and they add or delete more files than intended. I just noticed today that a file was mysteriously missing, and I know at one point there had been a few too many blank files next to it. It's likely I deleted the extra blank files and unintentionally deleted the other one as well without noticing at the time. 

The undo button doesn't work on file addition or deletion, and it's too easy to delete files unintentionally. I'll be more careful from now on, but it would be nice to have some safeguard against accidental file deletion.

Thank you for making such awesome software, though. I love Manuskript!

**Comment by TheJackiMonster** [comment]
Yes, that's a good point. I think the undo functionality should act consistently in those cases. So this needs to improve.

Using git for versioning is a nice workaround though currently.
